docker/centos-graal-maven-s2i/s2i/assemble

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# This POC script is based on the openshift/source-to-image project documentation,
+# and loosely inspired by fabric8io-images/s2i's assemble script, but **MUCH** simplified;
+# a TODO future PRODUCTION version of this would probably want to re-use that script...
+
+set -ex
+
+cd /tmp/src/
+mvn package -Pnative -DskipTests -Dmaven.javadoc.skip=true -Dmaven.site.skip=true -Dmaven.source.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Dpmd.skip=true -Dfabric8.skip=true -e -B
```

Comment:
That's a lot of skip :-) But I'm all good with that.

Reply:
Maybe you can use multiline as the `-e -B` are quite important (and hidden right now)

Reply:
FYI The skip are [are stolen from here](https://github.com/fabric8io-images/s2i/blob/master/java/templates/s2i/assemble#L123). I've moved `-e -B` to the front and made it multi line.